test/event/handlers/DiscordMessageLinkHandlerTest.ts

```diff
@@ -25,9 +25,22 @@ describe("DiscordMessageLinkHandler", () => {
 			handler = new DiscordMessageLinkHandler();
 		});
 
+		it("sends a message in message channel when contains discord message link mid sentence", async () => {
+			const message = CustomMocks.getMessage();
+			const channel = CustomMocks.getTextChannel();
```

Comment:
Since you're declearing these in each `it` you could just do it in the `before`.